modules/lima/main.nf

```diff
@@ -0,0 +1,50 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process LIMA {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::lima=2.2.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/lima:2.2.0--h9ee0642_0"
+    } else {
+        container "quay.io/biocontainers/lima:2.2.0--h9ee0642_0"
+    }
+
+    input:
+    tuple val(meta), path(ccs)
+    path(primers)
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
+    path("*.bam.pbi"),              emit: pbi
+    path("*.xml"),                  emit: xml
+    path("*.json"),                 emit: json
+    path("*.clips"),                emit: clips
+    path("*.counts"),               emit: counts
+    path("*.guess"),                emit: guess
+    path("*.report"),               emit: report
+    path("*.summary"),              emit: summary
+    path("*.version.txt"),          emit: version
+
+    script:
+    def software = getSoftwareName(task.process)
+    def lima_out = ccs.toString().replaceAll(/bam$/, 'fl.bam')
+    """
+    lima \\
+        $ccs \\
+        $primers \\
+        $lima_out \\
+        -j $task.cpus \\
+        $options.args
+
+    lima --version | grep -e 'commit' > ${software}.version.txt
```

Comment:
You should strip away the rest so only the version number remains I think.

Reply:
This is fixed